python/cog.py

```diff
@@ -305,6 +348,28 @@ async def on_component(self, ctx: ComponentContext):
             embed.set_field_at(index=0, name=name, value=jump_url)
             await ctx.edit_origin(embed=embed)
 
+    async def send_files_as_script(self, ctx:SlashContext, files: List[Dict]):
+        """
+        Send a list of Discord files as a script to download them.
+
+        Args:
+            ctx: The originating context.
+            files: The files to send to the context.
+        """
+
+        script_top = export_template[0:EXP_TEMPLATE_SPLIT_IDX]
+        dict_strs = list(map(lambda d: str(d) + ',', files))
+        script_bottom = export_template[EXP_TEMPLATE_SPLIT_IDX:]
+        script_lines = []
+        script_lines.extend(script_top)
```

Comment:
I haven't seen that syntax before, but a quick SO search and playing in the REPL taught me we can just do `script_lines = script_top + dict_strs + script_bottom` and it will spread the elements instead of making a 2D array.